File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/SuccessfulPaymentFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.payments.SuccessfulPayment;

public class SuccessfulPaymentFilterValidator extends FilterValidator<SuccessfulPayment> {

    public SuccessfulPaymentFilterValidator(CustomValidator<SuccessfulPayment> validator) {
        super(SuccessfulPayment.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/SuccessfulPaymentFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.payments.SuccessfulPayment
import io.github.oleksivio.tl.kbot.core.core.controller.handler.check.Validator

class SuccessfulPaymentFilterValidator(validator: Validator<SuccessfulPayment>) : FilterValidator<SuccessfulPayment>(
    SuccessfulPayment::class, validator)
